2021-01-22-qubes-4-0-4-rc2.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "Qubes OS 4.0.4-rc2 has been released!"
+categories: releases
+download_url: /downloads/
+author: Andrew David Wong
+---
+
+We're pleased to announce the second release candidate for Qubes OS
+4.0.4.
+
+Qubes OS 4.0.4-rc2 includes many updates over the initial 4.0 release,
+in particular:
+
+- All 4.0 dom0 updates to date
+- Fedora 32 TemplateVM
+- Debian 10 TemplateVM
+- Whonix 15 Gateway and Workstation TemplateVMs
+- Linux kernel 5.4 by default
```

Comment:
@marmarek, is it 5.4 *by default*, or just as an option?

Reply:
Yes, it's the default one.